packages/client/src/runtime/core/engines/common/errors/engine-not-found/theseLocationsHaveBeenSearched.ts

```diff
@@ -0,0 +1,11 @@
+import { EngineNotFoundErrorInput } from './EngineNotFoundErrorInput'
+
+export function theseLocationsHaveBeenSearched(input: EngineNotFoundErrorInput) {
+  const { searchedLocations } = input
+
+  const uniqueSearchLocations = [...new Set(searchedLocations)]
+  const formattedSearchLocations = uniqueSearchLocations.map((location) => `  ${location}`).join('\n')
+
+  return `The following locations have been searched:
+${formattedSearchLocations}`
```

Comment:
Shouldn't we order the `formattedSearchLocations` paths somehow?

Reply:
I think we should leave the order as it has been searched intact just for ease of understanding when we read error reports.

Reply:
Ah, just saw this again https://github.com/prisma/prisma/pull/19330#discussion_r1199132788